client/scss/components/_footer.scss

```diff
@@ -166,6 +166,10 @@
       margin-inline-end: 50px;
     }
   }
+
+  &--no-margin {
+    margin-inline: 0;
+  }
```

Comment:
The footer assumes it's not in an element with `nice-padding` applied, as it has its own margin with nice-padding's values (see lines above this). With the way the generic views are set up, allow unsetting the margin so the footer actions do not have extra offset.

This will probably be made redundant with Thibaud's work on #9030 when that comes along.